.vscode/redbox.code-workspace

```diff
@@ -29,4 +29,12 @@
 			"path": ".."
 		}
 	],
+	"settings": {
+		"workbench.colorCustomizations": {
+			"activityBar.background": "#911212",
+			"activityBar.foreground": "#fbfbec",
+			"titleBar.activeBackground": "#911212",
+			"titleBar.activeForeground": "#fbfbec",
+		}
+	},
```

Comment:
I use a vscode plugin called [Window Colors](https://marketplace.visualstudio.com/items?itemName=stuart.unique-window-colors) to help me differentiate between projects. This makes redbox red